pkgs/development/libraries/tezos-rust-libs/default.nix

```diff
@@ -19,10 +19,11 @@ rustPlatform.buildRustPackage rec {
   '';
 
   postInstall = ''
-    mkdir $out/lib/tezos-rust-libs
     cp -r rustc-bls12-381/include $out/include
-    cp $out/lib/librustc_bls12_381.a $out/lib/tezos-rust-libs
-    cp $out/lib/librustzcash.a $out/lib/tezos-rust-libs
+    cp -r librustzcash/include $out
+    cp -r $out/lib $out/tmp
+    mkdir $out/lib/tezos-rust-libs
+    mv $out/tmp/ $out/lib/tezos-rust-libs/
```

Comment:
Can’t you use `opaline` or `dune install`?

Reply:
It's outside of the ocaml-packages so I'm not sure that makes sense?

Reply:
It does, particularly because the authors went through the trouble of writing a `.install` file that is precisely meant to be consumed by such a tool. However, said file seems very fragile so you manual post-install script is probably better.